tests/test_models.py

```diff
@@ -13,20 +15,27 @@
     solver = "cbc"
 
 TOL = 1e-5
+N_HOURS = 24
+BASELOAD = 2
 
-@pytest.fixture
-def technology_set():
+
+@pytest.fixture(scope="session")
+def technology_set_1():
+    """
+    This fixture uses creates technologies directly from
```

Comment:
```suggestion
    This fixture uses creates technologies directly from
```
which word do you want to use: uses or creates?

Reply:
Ah